azure-mgmt-compute/src/test/java/com/microsoft/azure/management/compute/VirtualMachineEncryptionOperationsTests.java

```diff
@@ -50,19 +53,19 @@ public void canEncryptVirtualMachine() {
                     .withNewPrimaryNetwork("10.0.0.0/28")
                     .withPrimaryPrivateIPAddressDynamic()
                     .withNewPrimaryPublicIPAddress(publicIpDnsLabel)
-                    .withLatestLinuxImage("RedHat", "RHEL", "7.2")
+                    .withPopularLinuxImage(KnownLinuxVirtualMachineImage.UBUNTU_SERVER_18_04_LTS)
                     .withRootUsername(uname)
                     .withRootPassword(password)
-                    .withSize(VirtualMachineSizeTypes.fromString("Standard_D2a_v4"))
+                    .withSize(VirtualMachineSizeTypes.STANDARD_DS3_V2)  // ADE need 8GB
```

Comment:
8GB or other settings? For future tests, it may refer [dav4-dasv4-series](https://docs.microsoft.com/en-us/azure/virtual-machines/dav4-dasv4-series).

Reply:
I will try Standard_D2a_v4

Reply:
OK Standard_D2a_v4 works.